recipes/mammos-mumag/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "mammos-mumag" %}
+{% set version = "0.10.1" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mammos_mumag-{{ version }}.tar.gz
+  sha256: 4ae75d7e9fea5aa6f8d5b3fb52631dadffb343683f56b889a84fea2f2fd681d7
+
+build:
+  entry_points:
+    - unv2fly = mammos_mumag.cli:convert_mesh
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - jinja2 >=3.1.6
+    - mammos-entity >=0.10.0
+    - mammos-units
+    - matplotlib-base
+    - numpy
+    - pandas
+    - psutil >=7
+    - pydantic >=2.11
+    - pyvista
+    - pyyaml >6
+    - requests
+    - scipy
+    - tqdm
+    - urllib3
+    - __linux  # [linux]
+    - __osx  # [osx]
+    - __win  # [win]
+    - jax  # [unix]
```

Comment:
Unless jax is optional, this should be vanilla noarch, and not OS-noarch.

It's okay if a noarch package fails on a specific platform due to a missing dependency. We prefer using noarch because it's faster to build, and the package will work as soon as the missing dependency becomes available on that platform.